cypress/components/help/help.cy.tsx

```diff
@@ -78,13 +63,16 @@ context("Tests for Help component", () => {
         "Warning: The tag <%s> is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.%s",
         asElem
       );
-      let errors;
+      let errors: any;
```

Comment:
To avoid using `any`, there is an alternative way we could write the test. What do you think about updating the test to be the following?:

```tsx
describe("check as prop for Help component", () => {
  it("should throw a console error when invalid html elements are used", () => {
    cy.spy(window.console, "error");
    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
    // @ts-ignore:next-line
    CypressMountWithProviders(<HelpComponent as="foo" />);
    cy.window().its("console.error").should("have.been.called");
  });
});
``` 

This should eliminate the need for you to type that `errors` object :)

Reply:
same here